local/config/0chain.yaml

```diff
@@ -0,0 +1,158 @@
+  view_change: true
+  smart_contract:
+    storage: true
+    faucet: true
+    zrc20: true
+    interest: true
+    miner: true
+    multisig: true
+    vesting: true
+  txn_generation:
+    wallets: 50
+    max_transactions: 0
+    max_txn_fee: 10000
+    min_txn_fee: 0
+    max_txn_value: 10000000000
+    min_txn_value: 100
+  faucet:
+    refill_amount: 1000000000000000
+
+zerochain:
+  id: "0afc093ffb509f059c55478bc1a60351cef7b4e9c008a53a6cc8241ca8617dfe"
+  decimals: 10
+  genesis_block:
+    id: "ed79cae70d439c11258236da1dfa6fc550f7cc569768304623e8fbd7d70efae4"
+
+server_chain:
+  id: "0afc093ffb509f059c55478bc1a60351cef7b4e9c008a53a6cc8241ca8617dfe"
+  owner: "edb90b850f2e7e7cbd0a1fa370fdcc5cd378ffbec95363a7bc0e5a98b8ba5759"
+  decimals: 10
+  tokens: 200000000
+  genesis_block:
+    id: "ed79cae70d439c11258236da1dfa6fc550f7cc569768304623e8fbd7d70efae4"
+  block:
+    min_block_size: 1
+    max_block_size: 10
+    max_byte_size: 1638400
+    generators: 2
+    replicators: 0
+    generation:
+      timeout: 15
+      retry_wait_time: 5 #milliseconds
+    proposal:
+      max_wait_time: 180 # milliseconds
+      wait_mode: static # static or dynamic
+    consensus:
+      threshold_by_count: 66 # percentage (registration)
+      threshold_by_stake: 0 # percent
+    sharding:
+      min_active_sharders: 25 # percentage
+      min_active_replicators: 25 # percentage
+    validation:
+      batch_size: 1000
+    reuse_txns: false
+    storage:
+      provider: blockstore.FSBlockStore # blockstore.FSBlockStore or blockstore.BlockDBStore
+  round_range: 10000000
+  round_timeouts:
+    softto_min: 3000 #in miliseconds
+    softto_mult: 3 #multiples of mean network time (mnt)  softto = max{softo_min, softto_mult * mnt}
+    round_restart_mult: 2 #number of soft timeouts before round is restarted
+    round_timeout_mult: 2 # increase timeout counter by this value
+  transaction:
+    payload:
+      max_size: 98304 # bytes
+    timeout: 30 # seconds
+    min_fee: 0
+  client:
+    signature_scheme: bls0chain # ed25519 or bls0chain
+    discover: true
+  messages:
+    verification_tickets_to: all_miners # generator or all_miners
+  state:
+    prune_below_count: 100 # rounds
+  smart_contract:
+    timeout: 8000 # milliseconds
+  health_check:
+    show_counters: true
+    deep_scan:
+      enabled: false
+      settle_secs: 30
+      window: 0 #Full scan till round 0
+      repeat_interval_mins: 3 #minutes
+      report_status_mins: 1 #minutes
+      batch_size: 50
+    proximity_scan:
+      enabled: true
+      settle_secs: 30
+      window: 100000 #number of blocks, Do not make 0 with minio ON, Should be less than minio old block round range
+      repeat_interval_mins: 1 #minutes
+      report_status_mins: 1 #minutes
+      batch_size: 50
+  lfb_ticket:
+    rebroadcast_timeout: "15s"  #
+    ahead: 5                    # should be >= 5
+    fb_fetching_lifetime: "10s" #
+  async_blocks_fetching:
+    max_simultaneous_from_miners: 100S
```

Comment:
is `S` a typo here?

Reply:
Yes. I only changed the magic_block_file entry.